content/doc/tutorials/create-a-pipeline-in-blue-ocean.adoc

```diff
@@ -8,430 +8,294 @@ section: doc
-. Fork the
-  https://github.com/jenkins-docs/creating-a-pipeline-in-blue-ocean[`creating-a-pipeline-in-blue-ocean`]
-  on GitHub into your local GitHub account. If you need help with this process,
-  refer to the https://help.github.com/articles/fork-a-repo/[Fork A Repo]
-  documentation on the GitHub website for more information. +
-  *Note:* This is a different repository to the one used in the
-  link:../build-a-node-js-and-react-app-with-npm[Build a Node.js and React app
-  with npm] tutorial. Although these repositories contain the same application
-  code, ensure you fork and use the correct one before continuing on.
-
-
-=== Create your Pipeline project in Blue Ocean
-
-. Go back to Jenkins and ensure you have accessed the Blue Ocean interface. To
-  do this, make sure you:
-* have browsed to `\http://localhost:8080/blue` and are logged in +
-  or
-* have browsed to `\http://localhost:8080/`, are logged in and have clicked *Open
-  Blue Ocean* on the left.
-. In the *Welcome to Jenkins* box at the center of the Blue Ocean interface,
-  click *Create a new Pipeline* to begin the Pipeline creation wizard. +
-  *Note:* If you don't see this box, click *New Pipeline* at the top right.
-. In *Where do you store your code?*, click *GitHub*.
-. In *Connect to GitHub*, click *Create an access key here*. This opens GitHub
-  in a new browser tab. +
-  *Note:* If you previously configured Blue Ocean to connect to GitHub using a
-  personal access token, then Blue Ocean takes you directly to step 9
-  <<choose-github-account,below>>.
-. In the new tab, sign in to your GitHub account (if necessary) and on the
-  GitHub *New Personal Access Token* page, specify a brief *Token description*
-  for your GitHub access token (e.g. `Blue Ocean`). +
-  *Note:* An access token is usually an alphanumeric string that respresents
-  your GitHub account along with permissions to access various GitHub features
-  and areas through your GitHub account. This access token will have the
-  appropriate permissions pre-selected, which Blue Ocean requires to access and
-  interact with your GitHub account.
-. Scroll down to the end of the page (leaving all other *Select scopes* options
-  with their default settings) and click *Generate token*.
-. On the resulting *Personal access tokens* page, copy your newly generated
-  access token.
+== Fork the sample repository on GitHub
+
+Fork the simple "Welcome to React" Node.js and React application on GitHub into your own GitHub account.
+
+. Ensure you are signed in to your GitHub account.
+If you don't yet have a GitHub account, sign up for a free on the https://github.com/[GitHub website].
+. Fork the https://github.com/jenkins-docs/creating-a-pipeline-in-blue-ocean[`creating-a-pipeline-in-blue-ocean`]
+on GitHub into your local GitHub account.
+If you need help with this process, refer to the https://help.github.com/articles/fork-a-repo/[Fork A Repo] documentation on the GitHub website for more information.
++
```

Comment:
I think you need an empty line before this note for it to format correctly. But it looks like it is displaying ok. So I am not sure if it needs this or not for sure.
```suggestion

+
```